packages/component-library-angular/src/document/Document.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { render } from '@testing-library/angular';
+import { clearElements } from '../utils';
+import { UtrechtDocument } from './component';
+
+import '@testing-library/jest-dom';
+
+afterEach(() => {
+  clearElements();
+});
+
+describe('Document', () => {
+  it('renders an HTML div element', async () => {
+    const { fixture } = await render(UtrechtDocument);
+
+    expect(fixture.nativeElement).toBeInTheDocument();
+  });
+
+  it('renders a design system BEM class name', async () => {
+    const { container } = await render(UtrechtDocument);
+
+    const doc = container.querySelector(':only-child');
+
+    expect(doc).toHaveClass('utrecht-document');
+  });
+
+  it('displays as CSS block element', async () => {
+    const { container } = await render(UtrechtDocument);
+
+    const doc = container.querySelector(':only-child');
+
+    expect(doc).toBeVisible();
+    expect(doc).toHaveStyle({ display: 'block' });
+  });
+
+  it('renders rich text content', async () => {
+    const fixture = TestBed.createComponent(UtrechtDocument);
+    const debugElement = fixture.debugElement.query(By.css('div'))!;
+    debugElement.nativeElement.innerHTML = '<h1>Breaking news</h1>';
+    fixture.detectChanges();
+    expect(debugElement.nativeElement).toContainHTML('h1');
+  });
```

Comment:
I don't think this approach for rendering rich text is a realistic test, we need to render it through Angular APIs. This test would still succeed even when we would remove `<ng-content/>`